lib/diaspora_federation.rb

```diff
@@ -22,10 +22,11 @@ module DiasporaFederation
     fetch_person_for_hcard
     save_person_after_webfinger
     fetch_private_key_by_id
-    fetch_private_key_by_post_guid
+    fetch_private_key_by_entity_guid
     fetch_public_key_by_id
-    fetch_public_key_by_post_guid
-    post_author_is_local?
+    fetch_public_key_by_entity_guid
```

Comment:
I think we should rename this to `fetch_author_public_key_by_entity_guid` to make it clearer (because its the author of the entity). And I also think that we should rename `fetch_public_key_by_id` to `fetch_public_key_by_diaspora_id` because id is to generic. (and the same for private key) What do you think?

and regardless of how good we name this callbacks, we should also document them somewhere (module-documentation of `DiasporaFederation`?). What the callbacks should do and what parameters they have.